literature.bib

```diff
@@ -5907,21 +6042,150 @@ @inproceedings{Q:SPLTea18
+	author    = {Nakanishi, Tsuneo and Hisazumi, Kenji and Fukuda, Akira},
+	title     = {{Teaching Software Product Lines as a Paradigm to Engineers: An Experience Report in Education Programs and Seminars for Senior Engineers in Japan}},
 	booktitle = SPLTea,
 	publisher = ACM,
-	address = NY,
-	location = {Gothenburg, Sweden},
-	pages = {46--47},
-	isbn = 9781450359450,
-	doi = {10.1145/3236405.3237204},
-	year = 2018,
-	numpages = 2
+	address   = NY,
+	location  = {Gothenburg, Sweden},
+	pages     = {46--47},
+	isbn      = 9781450359450,
+	doi       = {10.1145/3236405.3237204},
+	year      = 2018,
+	numpages  = 2,
+	ek-tags = {teaching}
+}
+
+@incollection{DHKL18,
+	author    = {Ferruccio Damiani and Reiner H\"{a}hnle and Eduard Kamburjan and Michael Lienhardt},
+	title     = {{Same Same But Different: Interoperability of Software Product Line Variants}},
+	booktitle = {Principled Software Development},
+	publisher = Springer,
+	address   = Cham,
+	pages     = {99--117},
+	doi       = {10.1007/978-3-319-98047-8_7},
+	year      = 2018
+}
+
+@inproceedings{KAS+:SPLC18,
+	author    = {Kr{\"{u}}ger, Jacob and Al-Hajjaji, Mustafa and Schulze, Sandro and Saake, Gunter and Leich, Thomas},
+	title     = {{Towards Automated Test Refactoring for Software Product Lines}},
+	booktitle = SPLC,
+	publisher = ACM,
+	address   = NY,
+	pages     = {143--148},
+	doi       = {10.1145/3233027.3233040},
+	year      = 2018,
+	month     = SEP
+}
+
+@inproceedings{KC:SPLC18,
+	author    = {Krueger, Charles and Clements, Paul},
+	title     = {{Feature-based Systems and Software Product Line Engineering With Gears From BigLever}},
+	booktitle = SPLC,
+	publisher = ACM,
+	address   = NY,
+	pages     = {1--4},
+	doi       = {10.1145/3236405.3236409},
+	year      = 2018,
+	month     = SEP,
+	ek-tags = {tooling}
+}
+
+@inproceedings{KES:SPLC18,
+	author    = {Krafczyk, Adam and El-Sharkawy, Sascha and Schmid, Klaus},
+	title     = {{Reverse Engineering Code Dependencies: Converting Integer-based Variability to Propositional Logic}},
+	booktitle = SPLC,
+	publisher = ACM,
+	address   = NY,
+	pages     = {34--41},
+	doi       = {10.1145/3236405.3237202},
+	year      = 2018,
+	month     = SEP,
+	ek-tags = {non-Boolean variability}
+}
+
+@inproceedings{KKH:SPLC18,
+	author    = {Kaindl, Hermann and Kramer, Stefan and Hoch, Ralph},
+	title     = {{An Inductive Learning Perspective on Automated Generation of Feature Models From Given Product Specifications}},
+	booktitle = SPLC,
+	publisher = ACM,
+	address   = NY,
+	pages     = {25--30},
+	doi       = {10.1145/3233027.3233031},
+	year      = 2018,
+	month     = SEP
+}
+
+@inproceedings{BZ:SPLC18,
+	author    = {Becker, Martin and Zhang, Bo},
+	title     = {{How Do Our Neighbours Do Product Line Engineering?: A Comparison of Hardware and Software Product Line Engineering Approaches From an Industrial Perspective}},
+	booktitle = SPLC,
+	publisher = {ACM Press},
```

Comment:
Interesting. This is different from ACM? Should we create a variable for that? Do the other SPLC18 entries also use this publisher?

Reply:
it was an oversight on my part, thanks for catching it